internal/server/router/router.go

```diff
@@ -12,17 +12,18 @@ import (
 
 	"github.com/openmeterio/openmeter/api"
 	"github.com/openmeterio/openmeter/internal/credit"
-	credit_httpdriver "github.com/openmeterio/openmeter/internal/credit/httpdriver"
+	creditdriver "github.com/openmeterio/openmeter/internal/credit/driver"
+	"github.com/openmeterio/openmeter/internal/credit/grant"
 	"github.com/openmeterio/openmeter/internal/debug"
 	debug_httpdriver "github.com/openmeterio/openmeter/internal/debug/httpdriver"
 	"github.com/openmeterio/openmeter/internal/entitlement"
-	entitlement_httpdriver "github.com/openmeterio/openmeter/internal/entitlement/httpdriver"
+	entitlement_httpdriver "github.com/openmeterio/openmeter/internal/entitlement/driver"
```

Comment:
please make the import names consistent in the whole package (e.g. use the same name), as this can be pretty bad if the package is imported with multiple aliases.

Reply:
> It's not in scope for this unit of work to consolidate all aliases by which these packages or resources referenced elsewhere. The goal is to set the direction going forward with the new naming, we can always iterate later.

I'm happy to fix these particular issues you highlighted but in general its not my intent